quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java

```diff
@@ -141,6 +144,21 @@ public Wall getWall(int x, int y, boolean isHorizontal) {
             return verticalWalls[x][y];
         }
     }
+    public List<Wall> getAllWall(){
+     List<Wall> wall= new ArrayList<>();
+        for (int y=0;y<height;y++){
+            for (int x=0;y<width;x++){
+                if(containsWall(x,y,true)){
```

Comment:
Add method and call once with `true` and once with `false`

Reply:
done, added addWallIfExists() method, by you *-: